docs/install/compose-jupy.yaml

```diff
@@ -0,0 +1,46 @@
+# docker run -it -p 8080:8080 -p 30040:8888 --rm -d --network mlrun-network --name jupyter
+# -v ${SHARED_DIR}:/home/jovyan/data mlrun/jupyter:1.0.0
+
+services:
+  jupyter:
+    image: "mlrun/jupyter:${TAG:-1.0.0}"
+    ports:
+      - "8080:8080"
+      - "8888:8888"
+    environment:
+      MLRUN_ARTIFACT_PATH: "/home/jovyan/data/{{project}}"
+      MLRUN_LOG_LEVEL: DEBUG
+      MLRUN_NUCLIO_DASHBOARD_URL: http://nuclio:8070
+      MLRUN_HTTPDB__DSN: "sqlite:////home/jovyan/data/mlrun.db?check_same_thread=false"
+      MLRUN_UI__URL: http://localhost:8060
+    volumes:
+      - "${SHARED_DIR:?err}:/home/jovyan/data"
+    networks:
+      - mlrun
+
+  mlrun-ui:
+    image: "mlrun/mlrun-ui:${TAG:-1.0.0}"
+    ports:
+      - "8060:80"
+    environment:
+      MLRUN_API_PROXY_URL: http://jupyter:8080
+      MLRUN_NUCLIO_MODE: enable
+      MLRUN_NUCLIO_API_URL: http://nuclio:8070
+      MLRUN_NUCLIO_UI_URL: http://localhost:8070
+    networks:
+      - mlrun
+
+  nuclio:
+    image: "quay.io/nuclio/dashboard:${NUCLIO_TAG:-stable-amd64}"
+    ports:
+      - "8070:8070"
+    environment:
+      NUCLIO_DASHBOARD_EXTERNAL_IP_ADDRESSES: "${HOST_IP:-127.0.0.1}"
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /tmp:/tmp
```

Comment:
nuclio can work without `/tmp`. you can omit that

Reply:
ok